pkg/operator/baremetal_config.go

```diff
@@ -6,13 +6,15 @@ import (
 	"net"
 
 	"github.com/golang/glog"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	dynamic "k8s.io/client-go/dynamic"
 )
 
 var provisioningGVR = schema.GroupVersionResource{Group: "metal3.io", Resource: "provisionings", Version: "v1alpha1"}
+var provisioningGR = schema.GroupResource{Group: "metal3.io", Resource: "provisionings"}
```

Comment:
IIRC, you can get this from `provisioningGVP.GroupResource()`, you could just use that where you've used this and not define the variable here (Though I'm not actually seeing where this is used?)

Reply:
Correct. This is left over from some earlier work. I'll remove it.